site/src/components/pages/user/tab-panel-contents-with-blocks.tsx

```diff
@@ -50,11 +51,10 @@ export const TabPanelContentsWithBlocks: FunctionComponent<
         }) => (
           <ListViewCard
             key={name}
-            type="block"
             icon={icon}
             title={displayName!}
             description={description}
-            lastUpdated={lastUpdated}
+            extraContent={formatUpdatedAt(lastUpdated)}
```

Comment:
We might as well reuse the `extraContent` for the blocks displayed here and in the `/blocks` route, unless there's some reason to hide the `version` and `author` here but not there.

Reply:
Makes sense, why not?
Addressed here https://github.com/blockprotocol/blockprotocol/pull/526/commits/f9968db0cde28c0704902c5535696e9e432e90b7